implementations/rust/ockam/ockam/src/entity/profile/rule.rs

```diff
@@ -0,0 +1,19 @@
+use super::*;
+
+#[derive(Clone, Debug)]
+pub struct ProfileRuleEd25519Verification {
+    public_key: Vec<u8>,
```

Comment:
Should we add id to each rule?

Reply:
That's a good question, I'm not sure.
Maybe we can use the index in the vector that contains the policies authoritative on which policy is being updated. 🤔

Reply:
@mrinal I think to simplify DID compliance we need to introduce string (human-readable or not) that would be id for VerificationMethod

Reply:
I was thinking something like this ... when serializing to a DID, we could serialize verification policy at index `1` to did doc with verification method:

```
{
  id: "did:ockam:1234"
  "verificationMethod": [{
      "id": "did:ockam:1234#verification_policy_1"
      ...
  }
}
```

either way, let's tackle it when we get to DID serialization of a profile.
for the time being I'm recording it in an issue https://github.com/ockam-network/ockam/issues/778